.github/scripts/install-torch-tensorrt.sh

```diff
@@ -1,13 +1,17 @@
-#!/usr/bin/env bash
 set -eou pipefail
-# Source conda so it's available to the script environment
-source ${BUILD_ENV_FILE}
-export EXTRA_INDEX_URL="https://download.pytorch.org/whl/nightly/${CU_VERSION}"
+
+INDEX_URL=https://download.pytorch.org/whl/${CHANNEL}/${CU_VERSION}
+PLATFORM=$(python -c "import sys; print(sys.platform)")
 
 # Install all the dependencies required for Torch-TensorRT
-${CONDA_RUN} pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated=legacy-resolver --extra-index-url=${EXTRA_INDEX_URL}
+pip install --pre "torch>=2.4.0.dev,<2.5.0" "torchvision>=0.19.0.dev,<0.20.0" --index-url ${INDEX_URL}
```

Comment:
It would be ideal to avoid the need to change the versions in this file as well if the requirements change. Could this instead reference versions in the `requirements.txt`, as below. and just override the `index-url`?

https://github.com/pytorch/TensorRT/blob/2a954f39a07e4def2298884e1b2cf45d86dd6c4b/py/requirements.txt#L5-L6